src/main.ts

```diff
@@ -185,12 +188,26 @@ const colorStops: HexColorString[][] = [
     "#ffe3aa",
     "#ffffff",
   ],
+  // Esri color ramps - Blue and Red 9
+  ["#d7191c", "#fdae61", "#ffffbf", "#abd9e9", "#2c7bb6"],
+  // Esri color ramps - Blue and Red 8
+  ["#ca0020", "#f4a582", "#f7f7f7", "#92c5de", "#0571b0"],
+  // Esri color ramps - Red and Green 9
+  ["#d7191c", "#fdae61", "#ffffbf", "#a6d96a", "#1a9641"],
+  // Esri color ramps - Purple and Red 2
+  ["#a53217", "#d2987f", "#fffee6", "#ab84a0", "#570959"],
+  // Esri color ramps - Green and Brown 1
+  ["#a6611a", "#dfc27d", "#f5f5f5", "#80cdc1", "#018571"],
 ];
 const colorRamps = colorStops.map((ramp) => new ColorRamp(ramp));
 const DEFAULT_RAMP = 4;
 
 function populateColorRampSelect(): void {
   colorRampSelectEl.innerHTML = "";
+  const width = 120, height = 25;
+  // Sets dimensions for color ramp container, as color ramp isn't inline (absolute/floating)
+  colorRampContainerEl.attributeStyleMap.set("width", `${width}px`);
+  colorRampContainerEl.attributeStyleMap.set("height", `${height}px`);
```

Comment:
This just sent me down a small rabbit hole about CSSOM - looks cool! ...but doesn't work in Firefox. \*sigh\* This is maybe more an argument to incorporate Babel into our toolchain than anything, but for now prefer `.style.width/height`.